### PR TITLE
Implemented bypassing malformed indicators

### PR DESCRIPTION
**Pull request by cyble-dev**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This is just a minor fix. Sometimes a few indicators fetched from our servers were malformed and were not properly structured. Due to this, the code was breaking. To fix this, we have configured to bypass such indicators.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by cyble-dev** [comment]
Hello @edik24, @Shellyber, @MLainer1
Please proceed, let me know if you need anything else.

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py [hunk 1]
```diff
@@ -164,7 +168,8 @@ def get_taxii(self, args: Dict[str, Any], is_first_fetch: bool = False):
                 elif response.get('ttps') or False:
                     content = response.get('ttps').get('ttps')
                 else:
-                    raise ValueError("Last fetch time retrieval failed.")
+                    continue
+                    # raise ValueError("Last fetch time retrieval failed.")
```
Comment (Shellyber) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:172]: If it is not needed, please remove :)

File: Packs/CybleThreatIntel/ReleaseNotes/2_0_23.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### Cyble Threat Intel
+- Implemented a check to ignore malformed indicators.
```
Comment (Shellyber) [inline Packs/CybleThreatIntel/ReleaseNotes/2_0_23.md:3]:
```suggestion
- Fixed an issue where the **fetch-indicators** command failed when encountering improperly structured indicators
```

**Review by Shellyber — CHANGES_REQUESTED** [review]
Nice work!
Please see my small comments.
Also, please have a look at the pre-commit step that fails, and fix it as well

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py (hunk 1 above)
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:172]: Done

File: Packs/CybleThreatIntel/ReleaseNotes/2_0_23.md (hunk 2 above)
Comment (cyble-dev) [reply Packs/CybleThreatIntel/ReleaseNotes/2_0_23.md:3]: Done

**Comment by cyble-dev** [comment]
Hi @Shellyber,
I have implemented the fixes suggested by you.
For the issue in pre-commit, I couldn't figure out how to fix it. I tried a couple of fixes but still getting the error. Please take a look at the error and commits and suggest how can I fix it.
Thanks.

**Comment by Shellyber** [comment]
@cyble-dev The error is happening because it doesn't get to the `except` part. 
The values for the try clause are:
val = []
time = 2022-06-11 00:00:00
It doesn't go the the exception, need to investigate further why

**Comment by cyble-dev** [comment]
Hi @Shellyber,
Fixed the test as well. Please proceed.

**Review by Shellyber — APPROVED** [review]
Great work!

**Comment by cyble-dev** [comment]
Hello @Shellyber,
Just noticed that the docker image is not update in release note. Somehow I missed it and it didn't reflect in test as well. Can you please update it?

**Pull request merged**